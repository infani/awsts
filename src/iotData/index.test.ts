import { IotData } from './index';

describe('index', () => {
  it('Publish', async () => {
    const res = await IotData.Publish('test', "test");
    expect(res.$metadata.httpStatusCode).toMatchSnapshot();
  });

  it('Client', async () => {
    const cli = await IotData.Client(
      './src/iotData/certificate.pem.crt',
      './src/iotData/private.pem.key',
      './src/iotData/AmazonRootCA1.pem',
      'a1whrgfehabj9-ats.iot.ap-northeast-1.amazonaws.com'
    );
    // await cli.connect()
    cli.publish("test", 'test2', 0)
  });
});
