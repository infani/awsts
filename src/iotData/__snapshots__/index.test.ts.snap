// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Publish 1`] = `200`;
